header.go

```diff
@@ -1793,6 +1793,69 @@ func (h *RequestHeader) peek(key []byte) []byte {
 	}
 }
 
+// PeekAll returns header value for the given key.
+func (h *RequestHeader) PeekAll(key string) [][]byte {
+	k := getHeaderKeyBytes(&h.bufKV, key, h.disableNormalizing)
```

Comment:
I wonder if we should do the same trick here as with `bufKV`. Where we put a `[][]byte` in `RequestHeader` and reuse that instead of allocating the return value on the heap.

Reply:
thanks,You're right

Reply:
I modified the logic and added a `mulHeader` filed.

Reply:
Can you add a comment like this to these functions?
```
// The returned value is not safe to use after the handler returns, the request is released or other calls to PeekAll.
```

Reply:
thanks